src/app/(app)/(admin)/admin/news/[id]/page.tsx

```
@@ -0,0 +1,239 @@
+
+            {/* Content */}
+            <div className="prose max-w-none">
+              <h2 className="text-xl font-semibold mb-4 text-gray-800">
+                Nội dung:
+              </h2>
+              <div
+                className="text-gray-700 leading-relaxed"
+                dangerouslySetInnerHTML={{
+                  __html: htmlContent || "Không có nội dung ",
+                }}
+              />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)